src/api/index.tsx

```diff
@@ -43,24 +43,37 @@ export async function getTransactionsPage(
     const response = await axios.get(
         `${getBaseUrl()}${baseEndpoint}/cblocks?limit=${0}&limit=${50}&blockStart=${startingBlock}&blockEnd=${endingBlock}&transactionId=${transactionId}`,
     )
-    return response.data.transactions.map(transaction => {
-        return {
-            blockNumber: parseInt(transaction.block),
-            transactionIndex: parseInt(transaction.index),
-            from: transaction.from,
-            hash: transaction.hash,
-            status:
-                parseInt(transaction.status) === 1
-                    ? 'Success'
-                    : `Failed-${parseInt(transaction.status)}`,
-            timestamp: parseInt(transaction.timestamp) * 1000,
-            to: transaction.to,
-            value: parseInt(transaction.value),
-            transactionCost: BigNumber(transaction.gasUsed)
-                .multipliedBy(BigNumber(transaction.effectiveGasPrice))
-                .toNumber(),
-        }
-    })
+    return response.data.transactions.map(
+        (transaction: {
+            block: string
```

Comment:
maybe it will be good if we have this type as an interface and use it instead of hardcode the transaction type each time